modules/react/toast/stories/toast.stories.mdx

```diff
@@ -36,44 +33,42 @@ purpose and provide an accessible experience.
 By default, `mode` is set to `status`, which indicates the message contains advisory information
 such as a successful action.
 
+
 <ExampleCodeBlock code={Basic} />
 
+
 A `status` `Toast` will wrap the message within a `polite` ARIA live region with a `role` of
 `status`.
 
-Here's a more complete example with a button which triggers a dismissable `Toast`. The `Toast` is
-positioned using `Popper`.
-
-<ExampleCodeBlock code={WithPopper} />
 
 ### Alert
 
 Set the `mode` prop to `alert` to convey urgent and important information such as an error.
 
 <ExampleCodeBlock code={ToastAlert} />
 
+
 An `alert` `Toast` will wrap the message within an `assertive` ARIA live region with a `role` of
 `alert`.
 
 ### Dialog
 
 Set the `mode` prop to `dialog` to convey the presence of a follow-up action. If you use this
-`mode`, you need to add an `aria-label`. This `aria-label` should be like additional information for the
+`mode`, you need to add an `aria-label`. This `aria-label` should be additional information for the
```

Comment:
Can we be more specific about what the `aria-label` should be set to? I saw it used to be hardcoded to "notification" for `mode="dialog"`, which was removed in #2044 ([link](https://github.com/Workday/canvas-kit/pull/2044/files#diff-4d15b085abe91101267e93a9b0a16ea872d6a79c5f97b27d1eaa401f8626b435L70)). Are there examples of other `aria-label` values that make sense in other contexts?